api/v1beta1/disruption_webhook.go

```diff
@@ -266,8 +274,14 @@ func safetyNetCountNotTooLarge(r Disruption) (bool, string, error) {
 	totalCount := 0
 	namespaceCount := 0
 	targetCount := 0
-	namespaceThreshold := 0.8
-	clusterThreshold := 0.66
+
+	if namespaceThreshold == 0 {
```

Comment:
We can directly remove these two `if` checks. 0 isn't a valid value, but if it we want it to be,it seems far more likely that the user means to disable these features (thus really means 100%). The defaulting at the CLI level is sufficient

Reply:
yeah thats a fair assumption, will remove.